examples/UNet/massachusetts_road_segm.py

```diff
@@ -52,12 +52,14 @@ def main():
     # (if you have, copy your repository including the data to an SSD, otherwise it will take a long time to
     # generate batches)
     mmap_mode = None
-    data_train = np.load("train_data.npy", mmap_mode=mmap_mode)
-    target_train = np.load("train_target.npy", mmap_mode=mmap_mode)
-    data_valid = np.load("valid_data.npy", mmap_mode=mmap_mode)
-    target_valid= np.load("valid_target.npy", mmap_mode=mmap_mode)
-    data_test = np.load("test_data.npy", mmap_mode=mmap_mode)
-    target_test = np.load("test_target.npy", mmap_mode=mmap_mode)
+    dataset = np.load("road_segm_dataset.npz", mmap_mode=mmap_mode)
```

Comment:
Memory-mapping only works for `.npy` files. The key idea is that the binary data of the `ndarray` is available as-is on disk, and can just be mapped into memory. This doesn't work when the binary data resides in a zip file.

How much smaller does the data get, though? What's the size of the `.npz` file?